content/shared/influxdb3-admin/backup-restore.md

```diff
@@ -0,0 +1,508 @@
+<!-- Allow leading shortcode -->
+{{% product-name %}} persists all data and metadata to object storage.
+Back up your data by copying object storage files in a specific order to ensure consistency and reliability.
+
+> [!Warning]
+> Currently, {{% product-name %}} does not include built-in backup and restore tools.
+> Because copying files during periods of activity is a transient process, we highly recommended you follow the below procedures and copy order to minimize risk of creating inconsistent backups.
+
+## Supported object storage
+
+InfluxDB 3 supports the following object storage backends for data persistence:
+
+- **File system** (local directory)
+- **AWS S3** and S3-compatible storage ([MinIO](/influxdb3/version/object-storage/minio/))
+- **Azure Blob Storage**
+- **Google Cloud Storage**
+ 
+> [!Note]
+> Backup and restore procedures don't apply to memory-based [object stores](/influxdb3/version/reference/config-options/#object-store).
+
+## File structure
+
```

Comment:
The table below outlines the file structure of your {{% product-name %}} object store.